packages/expo-secure-store/src/SecureStore.ts

```diff
@@ -176,6 +176,45 @@ export async function setItemAsync(
   await ExpoSecureStore.setValueWithKeyAsync(value, key, options);
 }
 
+/**
+ * Store a key–value pair synchronously.
+ * > It is not recommended to use this function with `requireAuthentication` set to `true`, as it will block the JavaScript
+ * thread until the user authenticates.
+ *
+ * @param key The key to associate with the stored value. Keys may contain alphanumeric characters
+ * `.`, `-`, and `_`.
+ * @param value The value to store. Size limit is 2048 bytes.
+ * @param options An [`SecureStoreOptions`](#securestoreoptions) object.
+ *
+ */
+export function setItemSync(key: string, value: string, options: SecureStoreOptions = {}): void {
+  _ensureValidKey(key);
+  if (!_isValidValue(value)) {
+    throw new Error(
+      `Invalid value provided to SecureStore. Values must be strings; consider JSON-encoding your values if they are serializable.`
+    );
+  }
+  if (!ExpoSecureStore.setValueWithKeySync) {
+    throw new UnavailabilityError('SecureStore', 'setItemSync');
+  }
+  return ExpoSecureStore.setValueWithKeySync(value, key, options);
+}
+
+/**
+ * Fetch the stored value associated with the provided key synchronously. This function can be used in the global scope.
+ * > It is not recommended to use this funciton with values which require authentication, as it will block the JavaScript
+ * thread until the user authenticates.
```

Comment:
```suggestion
 * Fetch the stored value associated with the provided key synchronously. This function can be used in the global scope.
 * > It is not recommended to use this function with values that require authentication, as it will block the JavaScript
 * thread until the user authenticates.
```